### PR TITLE
Remove more occurrences of "stex"

### PR DESCRIPTION
**Pull request by TechnoPorg**
A couple of places were missed when renaming `StreamTexture` to `CompressedTexture`. It's entirely possible that there are still places which I've missed, containing the old name.

### CODE REVIEW

**Comment by fire** [comment]
Did you get the class renames too?

File: scene/resources/texture.cpp [hunk 1]
```diff
@@ -2767,19 +2767,19 @@ CompressedTextureLayered::~CompressedTextureLayered() {
...
-		st = s;
-	} else if (p_path.get_extension().to_lower() == "scube") {
-		Ref<CompressedCubemap> s;
-		s.instantiate();
-		st = s;
-	} else if (p_path.get_extension().to_lower() == "scubearray") {
-		Ref<CompressedCubemapArray> s;
-		s.instantiate();
-		st = s;
+	Ref<CompressedTextureLayered> ct;
```
Comment (akien-mga) [inline scene/resources/texture.cpp:2770]: You missed some `st`'s below so it doesn't compile.

**Comment by akien-mga** [comment]
The `StreamTexture.svg` icon should likely be renamed to `CompressedTexture2D.svg`, currently I don't think anything would use it if it doesn't have `2D` in the name. It's supposed to be the icon for that specific type.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**